components/x-live-blog-post/src/LiveBlogPost.scss

```diff
@@ -74,3 +70,22 @@
 	border-radius: 50%;
 	background-color: oColorsByName('crimson');
 }
+
+.live-blog-post__bottom-controls {
+	display: flex;
+	justify-content: space-between;
+	align-items: center;
+	flex-wrap: wrap;
+	margin-top: oSpacingByName('s6');
+}
+
+.live-blog-post__bottom-controls .back-to-top {
```

Comment:
**Talking point:** It feels like some of these classes don't really align to BEM very well and I would like to see what other people think.

`live-blog-post__bottom-controls` I feel like the `bottom` part of this is not required info and if its styles were different based on its position then that should be a modifer. So I am thinking this should just be `.live-blog-post__controls`. What do you think?

`back-to-top` suggests that this is a new block level and that it is not tied to its parent block (the controls element) however the nesting also suggests that they are related.

I have a funny relation with nesting and BEM I can never quite decide on how to combine them both. I think a single level of nesting with BEM can be useful to make sure that relationship is clear and defined but also BEM should do this as well via its naming.

In this instance I think this class should be something more like `live-blog-post-controls__back-to-top`

What do others think?

Reply:
Yes, you're right. I'll fix them wiith your suggestions. Thanks 🙌🏽